scripts/build-release.ps1

```diff
@@ -23,17 +23,24 @@ function Build {
 
     # hashset key is package folder name in repo
 
-    $packages = @{ }
-    $packages["azure-iot-device"] = $env:device_version_part
-    $packages["azure-iot-nspkg"] = $env:nspkg_version_part
-    # TODO add new packages to this list
+    $packages = @{ } # TODO add new packages to this list
+
+    $packages["azure-iot-device"] = [PSCustomObject]@{
```

Comment:
Essentially what these changes in this file do is allow for different packages to have the package version maintained in a customizable location. Not all packages will need to scale up to having a `constant.py` type of file maintaining a single source of truth. Sometimes just maintaining within the `setup.py` will suffice.